server/lib/notifications.js

```diff
@@ -137,6 +141,61 @@ export async function notifyAdminsOfCollective(CollectiveId, activity, options =
   return notifySubscribers(adminUsers, activity, options);
 }
 
+async function notifyAdminsOfCollectiveAndCheckUserExpensesAmount(activity) {
+  try {
+    await notifyAdminsOfCollective(activity.data.collective.id, activity);
+    await notifyUserIfExpensesExceededMinimumFormAmount(activity);  
+  } catch (error) {
+    console.log(error);
+  }
+  
+}
+      
+async function notifyUserIfExpensesExceededMinimumFormAmount(activity) {
+  try {
+    const host = await models.Collective.findById(activity.data.host.id);
+    if (host.currency && host.currency !== 'USD') {
+      return;
+    }
+    // ideally change to consider the current fiscal year instead(Example: might be from
+    // march to february instead of from the beginning from january)
+    const beginningOfCurrentYear = new Date(new Date().getFullYear(), 0, 1);
+    const userExpenses = await models.Expense.findAll({
+      where: {
+        UserId: activity.data.user.id,
+        createdAt: { [Op.gte]: beginningOfCurrentYear },
+      }
+    });
+    let totalAmount = 0;
+    for (const expense of userExpenses) {
+      let expenseAmountInHostCurrency = expense.amount;
+      if (expense.currency !== host.currency) {
+        const expenseDate = (expense.status === expenseStatuses.PAID) ? expense.updatedAt : 'latest';
+        expenseAmountInHostCurrency = await convertToCurrency(expense.amount, expense.currency, host.currency, expenseDate);
+      }
+      totalAmount += expenseAmountInHostCurrency;
+    }
+    // U$ 600.00 total amount allowed without form as of July 2018
+    if (totalAmount > MAXIMUM_AMOUNT_TO_SEND_FORM_COMMENT) {
+      const fromCollectiveIdData = await models.Collective.findOne({ 
+        where: { 
+          name: OPENCOLLECTIVE_BOT_NAME
```

Comment:
I see why you wanted to have a constant with the name, but in that case I'd rather use `slug` or `id` which are unique (name is not); i.e. `IRS_BOT_COLLECTIVE_ID` or `IRS_BOT_COLLECTIVE_SLUG`